script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -688,7 +686,7 @@ I guess that's something else I'll have to start paying attention to starting to
 [sha:80e5f377531cc81eb0251309c7c0cbb7977a1f91]{
 -- Page 121, Offset 1470.
 -- 「なんだよ、反応薄いな。
-"Hey, what's with the lack of response?
+"You sure are taking this easy.
```

Comment:
Not sure taking it easy is the right phrase here, thin reaction seems more to me like he's out of it than relaxed, and makes more sense with the anaemia connection

Reply:
Thought it made a lot of sense in context, you know. He's being told all this stuff about murders and what not and barely reacting

Reply:
It just feels too intentional, like he has processed it and decided "yeah don't care lol", which doesn't lead well into the "I am barely clinging to consciousness because blood gone" assumption that underpins Arihiko's next line.